appveyor.yml

```diff
@@ -11,24 +11,28 @@ environment:
 
 build_script:
   - cmd: SET STRONG_NAME_OPTIONS=/p:SignAssembly=true;AssemblyOriginatorKeyFile=Confluent.Kafka.snk;PackageId=Confluent.Kafka.StrongName;Title=Confluent.Kafka.StrongName;AssemblyName=Confluent.Kafka.StrongName;BaseOutputPath=bin-sn\
+  - cmd: SET STRONG_NAME_OPTIONS_SR=/p:SignAssembly=true;AssemblyOriginatorKeyFile=Confluent.SchemaRegistry.snk;PackageId=Confluent.SchemaRegistry.StrongName;Title=Confluent.SchemaRegistry.StrongName;AssemblyName=Confluent.SchemaRegistry.StrongName;BaseOutputPath=bin-sn\
```

Comment:
what's with the trailing backslash?

Reply:
i don't know, it's probably not necessary, this is just copying from the Confluent.Kafka case which works.

not for review. there's a lot more to be done here. this is for testing only.